cli/internal/cache/async_cache.go

```diff
@@ -76,7 +76,7 @@ func (c *asyncCache) Shutdown() {
 // run implements the actual async logic.
 func (c *asyncCache) run() {
 	for r := range c.requests {
-		c.realCache.Put(r.anchor, r.key, r.duration, r.files)
+		_ = c.realCache.Put(r.anchor, r.key, r.duration, r.files)
```

Comment:
I'm doing this to silence the lint error. It doesn't make sense unless we have an error group here and that's something that cache-to-tar will obviate the need for.